docs/en/week08/08-3.md

```diff
@@ -0,0 +1,198 @@
+
+### Contractive Autoencoder
+
+A contractive autoencoder is an unsupervised deep learning technique that helps a neural network encode unlabeled training data. It makes this encoding less sensitive to small variations in its training dataset. This is accomplished by adding a regularizer, or penalty term, to whatever cost or objective function the algorithm is trying to minimize. The end result is to reduce the learned representation’s sensitivity towards the training input. 
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/contractiveAE.png" height="400px" /><br>
+<b>Fig. 3</b>: Contactive autoencoder
+</center>
+
+As shown in *Fig. 3*, $C(y, \tilde y)$ penalises sensitivities to reconstructions along the manifold while $R(h)$ penalises how much the wiggling of $h$ comes from the wiggling of $y$. In this manner, you basically get to push up the energies only in the direction that are not used for the reconstruction.
+
+### Variational Autoencoder (VAE)
+
+**Intuition behind VAE and a comparison with classic autoencoders**
+
+Next, we introduce Variational Autoencoders (or VAE), a type of generative model. But why do we even care about generative models? To answer the question, discriminative models learn to make predictions given some observations, but generative models aim to simulate the data generation process. One effect is that generative models can better understand the underlying causal relations which leads to better generalization.
+
+Note that although VAE has "Autoencoders" (AE) in its name (because of structural or architectural similarity to auto-encoders), the formulations between VAEs and AEs are very different. See *Fig. 4* below.
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/VAE.png" height="400px" /><br>
+<b>Fig. 4</b>: VAE vs. Basic AE
+</center>
+
+**What's the difference between variational autencoder (VAE) and basic autoencoder (AE)?**
+
+For VAE:
+- First, the encoder stage: we pass the input $\boldsymbol{x}$ to the encoder. Instead of generating a hidden representation $\boldsymbol{h}$ (the code) in AE, the code in VAE comprises two things: $\mathbb{U}(\boldsymbol{z})$ and $\mathbb{V}(\boldsymbol{z})$ where $\boldsymbol{z}$ is the latent random variable following a Gaussian distribution with mean $\mathbb{U}(\boldsymbol{z})$ and variance $\mathbb{V}(\boldsymbol{z})$. Note that people use Gaussian distributions as the encoded distribution in practice, but other distributions can be used as well.
```

Comment:
Please, don't use `\mathbb{}` for `U` and `V`.
`\mathbb{V}` means variance…

Reply:
fixed